dvc/fs/gdrive.py

```diff
@@ -296,6 +296,15 @@ def _with_bucket(self, path):
 
         return super()._with_bucket(path)
 
+    def _strip_bucket(self, entry):
```

Comment:
we have similar implementation in the `ObjectFSWrapper` ? what is the difference here?

Reply:
ObjectFSWrapper is for object-based filesystems, which has some shared methods that we need, especially for directories-like path `/dir/`, etc. It is copied from `ObjectFSWrapper` as you said. This is a temporary stuff that we need, until we get rid of `PathInfo`/`CloudURLInfo`.